addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/handler/MilightBridgeV6Handler.java

```diff
@@ -90,7 +90,12 @@ protected Runnable getKeepAliveRunnable() {
             @Override
             public void run() {
                 // logger.debug("Session keep alive");
-                session.keep_alive(refrehIntervalSec * 1000);
+                try {
+                    session.keep_alive(refrehIntervalSec * 1000);
```

Comment:
I'm sure there are more coding conventions violated. I've adapted and extended this binding when I wasn't aware of the guidelines. If possible I'd like to fix all in one in another PR.